src/firefly/js/ui/PositionFieldDef.js

```diff
@@ -68,6 +69,21 @@ export function formatTargetForHelp(wp) {
         '</div>';
 }
 
+function getEQJ2000(wp) {
```

Comment:
have you conform wp will never be `undefined` here?

Reply:
There's a null check for the 'wp' variable in the first line of the `formatPosForHelp` function @robyww, before the call to `getEQJ2000`. Should I also add a check for it being undefined at the top here?

Reply:
may just return an empty string

Reply:
@robyww I added a check for undefined from the call to `VisUtil.convert` (since it may return undefined) in my `getEQJ2000` function. And if I get the world point as undefined, then I just return an empty string like you suggested..tested forceful return of undefined to ensure it works. Merging now.